src/requireConfig.ts

```diff
@@ -0,0 +1,10 @@
+import { FirebaseError } from "./error";
+import { Options } from "./options";
+
+export function requireConfig(options: Options): void {
+  if (!options.config) {
+    throw options.configError
+      ? options.configError
+      : new FirebaseError("Not in a Firebase project directory (could not locate firebase.json)");
+  }
+}
```

Comment:
Looks like `requireConfig` is expected to be a promise since it's often used in commands' `before(requireConfig)` chain.

Reply:
Booo - I thought I checked in on this. There's absolutely no reason it _needs_ to be in a promise... I'll see if I can fix the underlying bit or make this actually return a promise